Automattic/pocket-casts-ios
# Bookmarks: Play the tone when a bookmark is created

| 📘 Part of: #1061 |
|:---:|

This plays a sound when adding a bookmark from the headphone action, if the user has the setting enabled. 

## To test

Note: Enable the bookmarks and patron feature flags.
Note 2: Since this requires testing of the headphone action you'll need to run this on a real device. 

1. Launch the app
2. Connect some headphones to your device
3. Go to Profile Tab > Cog > Headphone Controls
4. Enable the Add Bookmark option as one of the actions
5. When the Bookmark Confirmation Sound option appears, toggle it off, then back on
6. ✅ Verify the bookmark creation tone plays
7. Using your headphones perform the add bookmark action (triple or double click if using AirPods)
8. ✅ Verify the sound plays
9. Disable the Confirmation Sound option
10. Perform the headphone action again
11. ✅ Verify the sound does not play
12. Enable it again
13. Open the full screen player
14. Tap the Add Bookmark action
15. ✅ Verify the bookmark creation screen opens and the sound does not play
16. Connect to CarPlay
17. Tap the Next/Previous button
18. ✅ Verify the sound plays

## Checklist
